specification/personal-demographics.yaml

```diff
@@ -112,6 +112,10 @@ info:
 
     For more details, see [service levels](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#service-levels).
     
+    ## Rate limits
+
+    The default rate limit is 5TPS (Transactions Per Second), per app. If you require a higher rate limit please [contact us](https://digital.nhs.uk/developer/help-and-support). or raise this during the onboarding process.
```

Comment:
Is this true? Can users only send 5 TPS?

Reply:
[I believe so!](https://nhsd-jira.digital.nhs.uk/browse/SPINEDEM-2654) (AC1)

Reply:
Interesting. Thanks.